### PR TITLE
modify slider

### PR DESCRIPTION
**Pull request by LarryMatte**
rmv link and add addStyleNames

### CODE REVIEW

**Comment by christiangoudreau** [comment]
LGTM

File: src/main/java/com/arcbees/website/client/application/ApplicationView.ui.xml [hunk 1]
```diff
@@ -75,7 +75,7 @@
                         <a href="#{nameTokens.CONTACT}" class="{res.style.menuContact}">Contact</a>
                     </li>
                     <li>
-                        <a href="#{nameTokens.DEVELOPERS}">
+                        <a href="http://dev.arcbees.com/" target="_blank">
```
Comment (meriouma) [inline src/main/java/com/arcbees/website/client/application/ApplicationView.ui.xml]: Can we remove all the developers package (All the presenters / views nametokens)? Nothing shoud point to `/developers` now?

**Comment by meriouma** [comment]
LGTM, if we can remove the /developers stuff just tell me, I'll take care of it

**Comment by meriouma** [comment]
Can we also remove `/docs` stuff?

**Comment by LarryMatte** [comment]
@meriouma Yep we can rmv those...

**Pull request merged**